packages/office-ui-fabric-react/src/components/Dropdown/examples/Dropdown.Required.Example.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { Dropdown, DropdownMenuItemType } from 'office-ui-fabric-react/lib/Dropdown';
+import { BaseComponent } from 'office-ui-fabric-react/lib/Utilities';
+
+export class DropdownRequiredExample extends BaseComponent<{}, {}> {
+  constructor(props: {}) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    return (
+      <div className="docs-DropdownExample">
+        <Dropdown
+          placeholder="Select an Option"
+          label="Required dropdown example:"
+          id="Requireddrop1"
+          ariaLabel="Required dropdown example"
```

Comment:
Do we need an aria label on this if it's the exact same as the regular component label?

Reply:
Good point, we probably don't. Looks like we have this pattern in the other examples as well, just FYI.